datahub-web-react/src/app/search/ViewAllSearchItem.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import { Typography } from 'antd';
+import { SearchOutlined } from '@ant-design/icons';
+
+const ExploreForEntity = styled.span`
+    font-weight: light;
+    font-size: 16px;
+    padding: 5px 0;
+`;
+
+const ExploreForEntityText = styled.span`
+    margin-left: 10px;
+`;
+
+const ViewAllContainer = styled.div`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+`;
+
+const ReturnKey = styled(Typography.Text)`
+    & kbd {
+        border: none;
+    }
```

Comment:
what is `& kbd`? maybe having to do with `keyboard`?

Reply:
Antd's keyboard typography creates this html element which I just learned was a thing. Visually, I just wanted to remove the border because it looks a little better and matches the figma closer.

Maybe there's another way to select it without assuming it's a KBD element?

Reply:
ah yeah that's what i thought - I think this is safe! unfortunately we do just have to use ant design selectors sometimes. preferably not, but it's definitely unavoidable sometimes